resources/ext/ezbake.conf

```diff
@@ -16,6 +16,7 @@ ezbake: {
                # way to get rid of the hard-coded paths here, but I don't
                # know it.
                postinst: [
+                 "install --owner={{user}} --group={{user}} -d /opt/puppetlabs/server/data/puppetserver/jruby-gems",
```

Comment:
Did we not have this here in the first place because it wasn't necessary to have done at install time - i.e., because the `puppetserver gem` tool would create the directory automatically the first time it needed to be created and Puppet Server doesn't barf if the directory is set as the `GEM_HOME` but not present when it is started?